src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -712,6 +712,9 @@ export default class ConvasController {
         if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
             return;
         }
```

Comment:
@erikquinn do you know what I'm talking about here?  I'm not near a computer and can't type out some examples.